sw-precache-config.js

```diff
@@ -1,11 +1,9 @@
 module.exports = {
   staticFileGlobs: [
     '/manifest.json',
-    '/bower_components/webcomponentsjs/webcomponents-lite.min.js',
+    '/bower_components/webcomponentsjs/*.js',
```

Comment:
I know you didn't do that but do you mind updating this to only have `'/bower_components/webcomponentsjs/webcomponents-loader.js',` and use runtimeCache for other polyfill files. See how we did that in Shop: https://github.com/Polymer/shop/blob/app-engine-node/sw-precache-config.js#L5

Reply:
Oh! I'm going to do the same in the PSK:
[PolymerElements/polymer-starter-kit / sw-precache-config.js](https://github.com/PolymerElements/polymer-starter-kit/blob/master/sw-precache-config.js)

About the `index.html`, should it be in `staticFileGlobs` or in `navigateFallback`? Or in both? Or in none?

- news: in none.
- shop: in `staticFileGlobs`.
- polymer-starter-kit: in `staticFileGlobs` & `navigateFallback`.

Reply:
Technically, `index.html` is not needed in either `staticFileGlobs` or `navigateFallback` since `polymer build` will already do so. Including it is a no-op.